instrumentation/src/main/java/io/opentelemetry/android/OtelAndroidConfig.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.android;
+
+import io.opentelemetry.android.instrumentation.network.CurrentNetworkProvider;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.common.AttributesBuilder;
+
+/**
+ * Configuration object for OpenTelemetry Android. The configuration items in this class will be
+ * used in the OpenTelemetryRumBuilder to wire up and enable/disable various mobile instrumentation
+ * components.
+ */
+public class OtelAndroidConfig {
```

Comment:
I believe that maybe the name of this class should somehow mention where it belongs, so maybe something like `OtelRumConfig` or `OpenTelemetryRumConfig` or maybe rename `OpenTelemetryRum` to `OpenTelemetryAndroid`? naming is hard 😅

Reply:
I think OtelRumConfig is a good change. 👍🏻